crowdfunding_compassion/models/crowdfunding_project.py

```diff
@@ -68,7 +68,7 @@ class CrowdfundingProject(models.Model):
         compute="_compute_sponsorships"
     )
     invoice_line_ids = fields.One2many(
-        "account.invoice.line", "crowdfunding_project_id", string="Donations"
+        "account.invoice.line", "crowdfunding_participant_id", string="Donations"
```

Comment:
This won't work because the inverse field `crowdfunding_participant_id` points to a participant and not a project. It will therefore link all donations of the participant that has the same id as the project which is wrong! The only possible way to have this relation if the inverse Man2one doesn't exist anymore is to do a compute field.